examples/next/faust-nx/wp-templates/archive.js

```diff
@@ -0,0 +1,72 @@
+import { gql } from '@apollo/client';
+import * as MENUS from '../constants/menus';
+import { BlogInfoFragment } from '../fragments/GeneralSettings';
+import {
+  Header,
+  Footer,
+  Main,
+  Container,
+  EntryHeader,
+  NavigationMenu
+} from '../components';
+
+function Component(props) {
+  const { title: siteTitle, description: siteDescription } = props?.data?.generalSettings;
+  const primaryMenu = props?.data?.headerMenuItems?.nodes ?? [];
+  const footerMenu = props?.data?.footerMenuItems?.nodes ?? [];
+  const { name } = props.data.nodeByUri;
+
+  return (
+    <>
+      <Header
+        title={siteTitle}
+        description={siteDescription}
+        menuItems={primaryMenu}
+      />
+      <Main>
+        <>
+          <EntryHeader title={`Category: ${name}`} />
```

Comment:
Since the `archive` template could also be responsible for tags (i.e. `/tags/apple`), we may want to either rename this file to `category` or make the current `archive` page work with any type of archive.